lib/sourceParams.js

```diff
@@ -37,6 +37,16 @@ module.exports = function(app) {
     multipart: params.multipart || pkg.rooseveltConfig.multipart || {'multiples': true},
     maxLagPerRequest: params.maxLagPerRequest || pkg.maxLagPerRequest || 2000,
     shutdownTimeout: params.shutdownTimeout || pkg.shutdownTimeout || 30000,
+    // https behavior - generally no defaults, user-defined
+    https: params.https || pkg.rooseveltConfig.https || false,
+    httpsOnly: params.httpsOnly || pkg.rooseveltConfig.httpsOnly || false,
+    httpsPort: params.httpsPort || pkg.rooseveltConfig.httpsPort || 43733,
+    pfx: params.pfx || pkg.rooseveltConfig.pfx || false,
+    keyPath: params.keyPath || pkg.rooseveltConfig.keyPath, // object with pfx / key+cert (file paths)
+    passphrase: params.passphrase || pkg.rooseveltConfig.passphrase, // string
+    ca: params.ca || pkg.rooseveltConfig.ca, // string or array of strings (file paths)
```

Comment:
Could you add an additional OR so that `keyPath`, `passphrase`, and `ca` will be `null` if the `params` or Roosevelt config are undefined. Just to keep it consistent with the defaults specified in your documentation.